### ISSUE TITLE
CRON output via email from wall messages

### ISSUE THREAD

**Issue by billglick** [ncsa/puppet-profile_update_os#14]
When `wall` runs from CRON it outputs the following warning:
```
wall: cannot get tty name: Inappropriate ioctl for device
```

By default, `wall` is doing this because the TTY name is in the banner message, e.g.
```
Broadcast message from user@host (pts/0)
```
...but there is no TTY when running from CRON.

The solution is to use the `-n` option for `wall` to _suppress the banner_:
```
0 5 * * * ( /root/cron_scripts/run-if-today.sh 2 Wed && /usr/bin/wall -n 'This server (asd-pup.internal.ncsa.edu) will be updated and rebooted in 1 hour.' )
```

**Comment by billglick** [comment]
The following need to be updated:
- https://github.com/ncsa/puppet-profile_update_os/blob/22128f8267f93b0f464e49e1170949cc85585858/files/root/cron_scripts/kernel_upgrade.sh#L86
- https://github.com/ncsa/puppet-profile_update_os/blob/22128f8267f93b0f464e49e1170949cc85585858/files/root/cron_scripts/kernel_upgrade.sh#L92
- https://github.com/ncsa/puppet-profile_update_os/blob/22128f8267f93b0f464e49e1170949cc85585858/files/root/cron_scripts/kernel_upgrade.sh#L120
- https://github.com/ncsa/puppet-profile_update_os/blob/22128f8267f93b0f464e49e1170949cc85585858/files/root/cron_scripts/kernel_upgrade.sh#L128
- https://github.com/ncsa/puppet-profile_update_os/blob/1ddf82231503dd7ad829264925a8b05ac1279201/files/root/cron_scripts/run-if-today.sh#L5
- https://github.com/ncsa/puppet-profile_update_os/blob/1ddf82231503dd7ad829264925a8b05ac1279201/files/root/cron_scripts/run-if-today.sh#L6
- https://github.com/ncsa/puppet-profile_update_os/blob/1ddf82231503dd7ad829264925a8b05ac1279201/files/root/cron_scripts/run-if-today.sh#L7
- https://github.com/ncsa/puppet-profile_update_os/blob/2c0af2a7f2c0e7a52b796f36fb03300ab829408f/manifests/kernel_upgrade.pp#L90
- https://github.com/ncsa/puppet-profile_update_os/blob/2c0af2a7f2c0e7a52b796f36fb03300ab829408f/manifests/kernel_upgrade.pp#L94
- https://github.com/ncsa/puppet-profile_update_os/blob/2c0af2a7f2c0e7a52b796f36fb03300ab829408f/manifests/kernel_upgrade.pp#L99

**Comment by bsper2** [comment]
This was fixed in #13
